code/modules/clothing/glasses/glasses.dm

```diff
@@ -318,13 +318,21 @@
 	chameleon_action.emp_randomise()
 
 /obj/item/clothing/glasses/thermal/monocle
-	name = "Thermoncle"
-	desc = "A monocle thermal."
+	name = "thermoncle"
+	desc = "Never before has seeing through walls felt so gentlepersonly."
 	icon_state = "thermoncle"
 	flags = null //doesn't protect eyes because it's a monocle, duh
 
+/obj/item/clothing/glasses/thermal/monocle/examine(mob/user) //Different examiners see a different description!
+	if(user.gender == MALE)
+		desc = replacetext(desc, "person", "man")
+	else if(user.gender == FEMALE)
+		desc = replacetext(desc, "person", "woman")
+	..()
+	desc = initial(desc)

```

Comment:
You should cache the value of desc at the start of the proc instead of using `initial()`